thirdparty/cli-utils/status/cmdstatus.go

```diff
@@ -128,7 +128,7 @@ func (r *Runner) runE(c *cobra.Command, args []string) error {
 		return err
 	}
 
-	invInfo, err := live.ToInventoryInfo(inv)
+	invInfo, err := live.ToInventoryInfo(inv, inventory.NameStrategy)
```

Comment:
I don't like that this has to be set as default in every command. Is there a way to set it as default in the struct so that it uses the NameStrategy if strategy is not set?

Reply:
Refactored and no longer set in every command.